en/docs/learn/integration-tutorials/sending-a-simple-message-to-a-datasource.md

```diff
@@ -1,43 +1,50 @@
-# How to Expose a Datasource as a Service
+# Build your first Data Service
```

Comment:
Isn't it better to have How to Expose a Datasource as a Service. As that explains why you need to create a dataservice.

Reply:
Since we have another example as [How to Expose an RDBMS Datasource](https://mi.docs.wso2.com/en/latest/learn/examples/data-integration/rdbms-data-service/) and this guide aims new users changed this to `Build your first Data Service`

Reply:
No, my concern is by looking at the topic, the need for creating a dataservice is not addressed. The question is why I should create my first dataservice.

Reply:
Fixed with 37951df